script.js

```diff
@@ -0,0 +1,36 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export default function () {
+  let choice = Math.random();
+  //get request:
+  if (choice >= .05) {
+    http.get(`http://localhost:3000/api/productOptions/products/${Math.floor(Math.random()*10000000)}`);
+    sleep(1);
+  }
+
+  //post review request:
+  else if (choice < .05) {
+    let url = `http://localhost:3000/api/productOptions/products/reviews/${Math.floor(Math.random()*10000000)}`
```

Comment:
This looks good! I see you are randomly picking a product from 0 to 10M. Do you think your testing would yield slightly different results if you were to favor the items at the end of your table(s)? Like from 9M to 10M?